kiosk/src/Components/MainMenu.tsx

```diff
@@ -4,35 +4,50 @@ import AddGameButton from "./AddGameButton";
 import GameList from "./GameList";
 import configData from "../config.json"
 import HighScoresList from "./HighScoresList";
+import { DeleteButton } from "./DeleteButton";
 import { tickEvent } from "../browserUtils";
+import DeletionModal from "./DeletionModal";
 
 interface IProps {
     kiosk: Kiosk
   }
 
 const MainMenu: React.FC<IProps> = ({ kiosk }) => {
-    const [buttonSelected, setButtonState] = useState(false);
+    const [addButtonSelected, setAddButtonState] = useState(false);
```

Comment:
Can react store enum (or enum-like) values in state? Not necessarily needed in this PR, but as this grows, I suspect it may become easier to have a single enum value for "selectedItem" that changes based on what's selected, rather than tracking booleans for every selectable item.

Reply:
I would have to look into that. 

I agree, I'm not a fan of how I'm dealing with knowing which component is selected. I was even thinking that using contexts for this would be better, to just give a global state for handling controls and the like. However, I don't have any experience with contexts, and even when I looked at how multiplayer does contexts and dispatching, I had a hard time grasping it. 

I wrote kiosk in the way that I currently know React, which is very limited and novice. This is something that I am hoping to improve quickly. If kiosk ends up being a hit/having a lot of users, I think the first step is to refactor it into a proper React-style app, along with porting it into pxt.

Reply:
Hmm I'm not sure either, but it sounds like it'd be a larger change. And all just food for thought at the moment. Not something you necessarily need right now.